scp.sh

```
@@ -0,0 +1,3 @@
+ssh edu-gpu "cd ~/temp/poc && sudo docker compose down"
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)